usr/lib/linuxmint/mintUpdate/kernelwindow.py

```diff
@@ -430,19 +478,12 @@ def __init__(self, application):
         else:
             self.main_stack.set_visible_child(info_box)
 
-        # Center on main window
-        window_size = self.window.get_size()
-        parent_size = self.application.window.get_size()
-        parent_position = self.application.window.get_position()
-        parent_center_x = parent_position.root_x + parent_size.width / 2
-        parent_center_y = parent_position.root_y + parent_size.height / 2
-        self.window.move(parent_center_x - window_size.width / 2,
-                         parent_center_y - window_size.height / 2)
-
```

Comment:
Wasn't that added in a previous commit?

Reply:
It's been moved, not removed. My local branch had it in a different spot, so I just moved it there without losing sleep over why positions had changed. My local branch is authoritative for me.